routers/review.router.js

```diff
@@ -1,9 +1,19 @@
 const reviewController = require('../controllers/review.controller').reviewController;
 const router = require('express').Router({mergeParams:true});
 const auth = require('../config/auth').auth;
+const _ = require('lodash');
```

Comment:
lodash is a library. We define it const _ to use lodash. 
The underscore seems weird; does the underscore serve a special purpose?